JitsiTrackError.js

```diff
@@ -29,109 +29,111 @@ TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.TRACK_MUTE_UNMUTE_IN_PROGRESS]
 TRACK_ERROR_TO_MESSAGE_MAP[JitsiTrackErrors.NO_DATA_FROM_SOURCE]
     = "The track has stopped receiving data from it's source";
 
+
+// FIXME: Using prototype inheritance because otherwise instanceof is not
```

Comment:
No, I got the code wrong and thought that we will not longer extend Error - it's ok as it is.